src/components/CytoscapeGraph/CytoscapeGraph.tsx

```diff
@@ -77,18 +77,14 @@ export class CytoscapeGraph extends React.Component<CytoscapeGraphProps, Cytosca
   }
 
   componentDidMount() {
-    console.log('CY: mounting graph component');
```

Comment:
I find them useful when working on the graph. I suppose we could add in the future something like this:
https://github.com/mreuvers/typescript-logging

Reply:
Yes, I agree. Some logs can be useful in certain situation but annoying the rest of the time. Categorising logs + changing level locally would help to satisfy both situations.